easybuild/tools/systemtools.py

```diff
@@ -105,6 +106,25 @@ def get_core_count():
     """NO LONGER SUPPORTED: use get_avail_core_count() instead"""
     _log.nosupport("get_core_count() is replaced by get_avail_core_count()", '2.0')
 
+def get_total_memory():
+
+    # Return total memory as an integer, a number of megabytes.
+
+    memtotal = None
+    os_type = get_os_type()
+
+    if os_type == LINUX and os.path.exists(PROC_MEMINFO_FP):
+        with open(PROC_MEMINFO_FP, 'r') as meminfo:
+            for memline in meminfo:
+                memline_sub = re.sub(r'^MemTotal:\s*(\d+)\s*kB$', r'\1', memline)
```

Comment:
yeah, this is a bit strange indeed

should be something like:

```python
res = re.search(r'^MemTotal:\s*(\d+)\s*kB$', memline)
if res:
    memtotal = int(res.group(1)) / 1024
```

but do test this, of course